src/main/resources/alfio/db/PGSQL/afterMigrate__006_VIEW_reservation_and_ticket_and_tx.sql

```diff
@@ -101,7 +101,7 @@ create view reservation_and_ticket_and_tx as (select
 
 from tickets_reservation
 left outer join ticket on tickets_reservation.id = ticket.tickets_reservation_id
-left outer join b_transaction on ticket.tickets_reservation_id = b_transaction.reservation_id
+left outer join (select * from b_transaction where status <> 'INVALID' ) as b_transaction on ticket.tickets_reservation_id = b_transaction.reservation_id
```

Comment:
`left outer join (select * from b_transaction where status <> 'INVALID' ) as b_transaction on ticket.tickets_reservation_id = b_transaction.reservation_id`

can be written as 

`left outer join b_transaction on ticket.tickets_reservation_id = b_transaction.reservation_id and b_transaction.status <> 'INVALID`

I think?